client/src/pages/Feed.tsx

```diff
@@ -0,0 +1,18 @@
+import { Post } from '../components/community/post/index';
+import { MockPosts } from '../components/community/post/MockPosts';
+import { Box } from '@chakra-ui/react';
+
+const style = {
+  backgroundColor: 'gray.50',
+};
+
+const Feed = () => {
+  return (
+    <Box width="100%" height="100%" overflow="auto" backgroundColor="blackAlpha.50" paddingX="30%">
```

Comment:
Great now use react query to link with backend. Populate your mongodb with some fake created post or add in manually